packages/functional-tests/tests/signin/relyingParties.spec.ts

```diff
@@ -2,131 +2,118 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { test, expect, newPagesForSync } from '../../lib/fixtures/standard';
+import { expect, test } from '../../lib/fixtures/standard';
 
 test.describe('severity-1 #smoke', () => {
   // Slowing down test, was timing out on credentials teardown
   test.slow();
 
-  // https://testrail.stage.mozaws.net/index.php?/cases/view/1293471
-  test('signin to sync and disconnect #1293471', async ({
+  test('signin to sync and disconnect', async ({
     target,
     credentials,
+    syncBrowserPages: { page, login, settings },
   }) => {
-    const { browser, page, login, settings } = await newPagesForSync(target);
-
+    test.fixme(true, 'Fix required as of 2024/04/19 (see FXA-9490)');
     await page.goto(
       target.contentServerUrl +
-        '?context=fx_desktop_v3&entrypoint=fxa%3Aenter_email&service=sync&action=email',
-      { waitUntil: 'load' }
+        '?context=fx_desktop_v3&entrypoint=fxa%3Aenter_email&service=sync&action=email'
     );
-
     await login.login(credentials.email, credentials.password);
 
     expect(await login.isSyncConnectedHeader()).toBe(true);
 
     // Normally we wouldn't need this delay, but because we will be
     // disconnecting the sync service, we need to ensure that the device
-    // record and web channels have been sent and created.
+    // record and web channels have been sent and created (FXA-9490).
     await page.waitForTimeout(1000);
```

Comment:
Just noting that this need for timeouts is an issue for all sync tests that wait for web channel events.